frontend/public/components/pod.tsx

```diff
@@ -889,6 +871,26 @@ export const PodsPage = connect<{}, PodPagePropsFromDispatch, PodPageProps>(
         ...listProps
       } = props;
       const { t } = useTranslation();
+
+      const filters = [
```

Comment:
Don't duplicate this code. Instead create a common function that returns these filters and uses `i18n.t`